README.Rmd

```diff
@@ -0,0 +1,153 @@
+<!-- README.md is generated from README.Rmd. Please edit that file -->
+<!-- BEWARE: THIS CODE DOESN'T RUN - IT'S JUST PSEUDO CODE FOR NOW -->
+
+```{r include=FALSE}
+knitr::opts_chunk$set(
+  comment = "#>",
+  collapse = TRUE,
+  warning = FALSE,
+  fig.path = "man/figures/README-",
+  out.width = "100%"
+)
+```
+
+# rcromwell
+
+<!-- badges: start -->
+[![Project Status: Experimental – Useable, some support, not open to feedback, unstable API.](https://getwilds.github.io/badges/badges/experimental.svg)](https://getwilds.github.io/badges/#experimental)
+[![R-CMD-check](https://github.com/getwilds/rcromwell/actions/workflows/R-CMD-check.yaml/badge.svg?branch=dev)](https://github.com/getwilds/rcromwell/actions/workflows/R-CMD-check.yaml)
+<!-- badges: end -->
+
+Convenience Tools for Managing WDL Workflows via Cromwell
+
+## Installation
+
+You can install the development version of `rcromwell` from [GitHub](https://github.com/) with:
+
+```r
+# install.packages("remotes")
+remotes::install_github("getwilds/rcromwell")
+```
+
+Install a specific release version (in this case v1.0) by:
+
+```r
+remotes::install_github('getwilds/rcromwell@v1.0')
+```
+
+## Set up your Cromwell Server
```

Comment:
We should remove this b/c it's FH specific.

Reply:
remove the set Cromwell URL step? but we need to set a URL for the package to work. Maybe you mean don't include any PROOF related stuff?

Reply:
The link to the DIY-cromwell-server is FH specific.  So remove this H2 and the stuff below it that is FH specific is all I meant. ;)